diff --git a/contrib/docker/Dockerfile b/contrib/docker/Dockerfile
index 7b03573..aa5c529 100644
--- a/contrib/docker/Dockerfile
+++ b/contrib/docker/Dockerfile
@@ -1,4 +1,4 @@
-FROM debian:bullseye as builder
+FROM debian:bullseye-slim as builder
 
 LABEL maintainer="Axel Gembe <derago@gmail.com>"
 
@@ -34,4 +34,4 @@ EXPOSE 50001 50002
 COPY contrib/docker/docker-entrypoint.sh /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
 
-CMD ["Fulcrum"]
+CMD ["Fulcrum", "/data/fulcrum.conf"]
diff --git a/contrib/docker/build.sh b/contrib/docker/build.sh
index ec82ed2..82e4a3c 100755
--- a/contrib/docker/build.sh
+++ b/contrib/docker/build.sh
@@ -13,4 +13,4 @@ here=$(dirname $(realpath "$0" 2> /dev/null || grealpath "$0"))
 . "$here"/../base.sh
 
 docker buildx build --build-arg MAKEFLAGS="-j $WORKER_COUNT" -t "$IMAGE_TAG" -f contrib/docker/Dockerfile \
-    --platform linux/arm64/v8,linux/amd64 --push "$here"/../..
+    --platform linux/amd64 --load "$here"/../..
diff --git a/contrib/docker/docker-entrypoint.sh b/contrib/docker/docker-entrypoint.sh
index 2fcbbd3..4af64bc 100755
--- a/contrib/docker/docker-entrypoint.sh
+++ b/contrib/docker/docker-entrypoint.sh
@@ -5,7 +5,7 @@ if [ ! -e "$SSL_CERTFILE" ] || [ ! -e "$SSL_KEYFILE" ] ; then
 fi
 
 if [ "$1" = "Fulcrum" ] ; then
-  set -- "$@" -D "$DATA_DIR" -c "$SSL_CERTFILE" -k "$SSL_KEYFILE"
+  set -- "$@" -c "$SSL_CERTFILE" -k "$SSL_KEYFILE"
 fi
 
 exec "$@"
